cli/src/main/java/Main.java

```diff
@@ -236,9 +239,25 @@ public static void allStatistics() {
         StatsProcessor stats = StatsProcessor.create();
         StatsData allStats = stats.getAllStats();
 
-        //Display aggregate statistics
-        System.out.println("\n- All Statistics -");
-        displayStatistics(allStats);
+        if (statisticalDataExists(allStats.getTotalRolls())) {
+            //Display aggregate statistics
+            System.out.println("\n- All Statistics -");
+            displayStatistics(allStats);
+        }
+    }
+
+    /**
+     * Displays an error if statistical data does not exist.
+     * i.e. Nobody has played the game yet.
+     * @param dataLength   Length of statistical data (total rolls or number of past players).
+     * @return             If the data equals zero (is empty).
+     */
+    public static boolean statisticalDataExists(int dataLength) {
```

Comment:
This method does 2 things:

1. Checks if the data exists
2. Prints an error message